pwiz_tools/Skyline/TestFunctional/TestFunctional.csproj

```diff
@@ -436,6 +436,7 @@
     <None Include="ConfigureToolsDlgTest.zip" />
     <None Include="CopyPasteTest.zip" />
     <None Include="CrosslinkingTest.zip" />
+    <None Include="DdaSearchTest.zip" />
```

Comment:
We could have saved some repo space by copying the file in the test rather than duplicating it in the zip file, but without rewriting the git history it's too late now. Not worth using BFG for 1.3MB.

Reply:
@brendanx67 What does including these zips in the project file accomplish? I'm wondering why something wasn't failing because it wasn't already included.

Reply:
It doesn't actually accomplish much, I guess, since we always keep them from being copied. I personally like to be able to see them in the project viewer, which is why I do it. I feel they really are part of the project. There is some utility in being able to turn on full file view in the Project viewer and consider files that are not added to the project as something worth looking at. Sometimes they turn out to be dead code, which should be removed from the Git source tree. If not adding things becomes normal, it is much harder to use this feature.